Mods/vcmi/config/vcmi/english.json

```diff
@@ -63,7 +63,11 @@
 	"vcmi.mainMenu.serverClosing" : "Closing...",
 	"vcmi.mainMenu.hostTCP" : "Host TCP/IP game",
 	"vcmi.mainMenu.joinTCP" : "Join TCP/IP game",
-	
+
+	"vcmi.multiplayerWelcomeMessage.additionalSettings" : "Additional settings",
+	"vcmi.multiplayerWelcomeMessage.cheatsAllowed" : "Cheats allowed",
+	"vcmi.multiplayerWelcomeMessage.unlimitedReplays" : "Unlimited battle replays",
```

Comment:
Why not reuse existing strings?
- vcmi.optionsTab.extraOptions.hover
- vcmi.optionsTab.cheatAllowed.hover
- vcmi.optionsTab.unlimitedReplay.hover